### PR TITLE
Docs/nb section3 supported platform Polaris

### PR DESCRIPTION
**Pull request by mtitov**

### CODE REVIEW

File: docs/source/supported/polaris.rst [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+  * 32 CPU cores
+
+    * Each core has 2 threads (``SMT=2``)
+
+  * 4 GPUs
+  * 512 GiB of memory
+
+.. note::
+
+   RADICAL-Pilot provides a possibility to manage the ``-l`` option for
```
Comment (andre-merzky) [inline docs/source/supported/polaris.rst:29]: Maybe add something like:

> [...] to manage the ``-l`` option (resource selection qualifier) for [...]

?

File: docs/source/supported/polaris.rst [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+**virtual environment with** ``conda``
+
+.. code-block:: bash
+
+   module load conda; conda activate
+   conda create -y -n ve.rp python=3.9
+   # OR clone base environment
+   #   conda create -y -p $HOME/ve.rp --clone $CONDA_PREFIX
+
+RADICAL-Pilot is installed after a corresponding virtual environment is
```
Comment (andre-merzky) [inline docs/source/supported/polaris.rst:69]: > RADICAL-Pilot is installed [...]

--> 

> Install RADICAL-Pilot after activating [...]

**Review by andre-merzky — APPROVED** [review]
Looks good to me - left minor comments.  Thanks!

File: docs/source/supported/polaris.rst (hunk 1 above)
Comment (mtitov) [reply docs/source/supported/polaris.rst]: done

File: docs/source/supported/polaris.rst (hunk 2 above)
Comment (mtitov) [reply docs/source/supported/polaris.rst]: done

File: docs/source/supported/polaris.rst [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+
+Run MongoDB instance
+^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: bash
+
+   # launch the server
+   $HOME/mongo/bin/mongod -f $HOME/mongo/etc/mongodb.polaris.conf
+   # shutdown the server
+   $HOME/mongo/bin/mongod -f $HOME/mongo/etc/mongodb.polaris.conf --shutdown
```
[the remark this answers is not in the document]
Comment (mtitov) [reply docs/source/supported/polaris.rst:129]: yeah, absolutely agree, added!

**Review by mturilli — APPROVED** [review]
LGTM

**Review by okilic1 — APPROVED** [review]

**Pull request merged**